amxmodx/CLang.cpp

```diff
@@ -12,6 +12,8 @@
 #include "CLang.h"
 #include "format.h"
 
+CVector<FILE *> FileList;
```

Comment:
Yep, this file has been cleaned up, and there was no reason to have such global var defined here when it's unrelated to file natives.